astropy/table/table.py

```diff
@@ -779,7 +779,7 @@ def show_in_browser(self,
 
     def pformat(self, max_lines=None, max_width=None, show_name=True,
                 show_unit=None, show_dtype=False, html=False, tableid=None,
-                align='right'):
+                tableclass=None, align='right'):
```

Comment:
Can you move the new `tableclass=None` to the end?  I know it makes more sense here, but in theory someone could be calling `pformat` with all positional arguments and so this will be an API break.